tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "module": "commonjs",
     "noEmit": true,
     "strict": true,
-    "target": "es5"
+    "target": "es5",
+    "skipLibCheck": true
```

Comment:
Had to add this to skip checking Joi types. Otherwise, build fails with the following error:

```
> tsc --build ./tsconfig.json

node_modules/@types/joi/index.d.ts:544:18 - error TS2430: Interface 'StringSchema<N>' incorrectly extends interface 'AnySchema'.
  Types of property 'required' are incompatible.
    Type '() => StringSchema<{ R: true; T: N["T"]; }>' is not assignable to type '() => this'.
      Type 'StringSchema<{ R: true; T: N["T"]; }>' is not assignable to type 'this'.

544 export interface StringSchema extends AnySchema {
```

Reply:
Thanks @mmiszy 
No problem I don't know how to solve this problem yet :sweat_smile: .